android/build.gradle

```diff
@@ -94,7 +94,7 @@ android {
 
     defaultConfig {
         applicationId "com.thebluealliance.androidclient"
-        minSdkVersion 16
+        minSdkVersion 17
```

Comment:
@phil-lopreiato any stats on users on API level 16?

Honestly I'd be in favor of going to 21 or 23 soon, but just curious on whether there's any drive to keep it lowerl

Reply:
If it is a must we can keep Android 16. I just used a dependency for the Sign In with Apple button so I could test really quickly. If we needed I can make a button just thought this would be easier lol. The dependency needed a minimum of API 17

Reply:
I've always just kept it where it was until we had a compelling enough reason to bump it. Bumping to 17 should have practically no effect